App/ProtoControllers/Groupe.php

```diff
@@ -37,12 +37,11 @@ public static function getOptions()
      /**
       * Retourne la liste des groupes de l'application
       *
-      * Il est fort probable que cette méthode change de portée. Pour le moment c'est pas utile
       *
       * @return array
       * @todo unescape_string ?
       */
-    private static function getListe()
+    public static function getListe()
```

Comment:
je dois mal m'y prendre, j'ai une erreur : 
`Fatal error: Constant expression contains invalid operations in App/ProtoControllers/Groupe.php on line 44`